src/states/main.js

```diff
@@ -97,6 +99,23 @@ export default class extends Phaser.State {
     });
   }
 
+  pause() {
```

Comment:
Should it be named `togglePause`, because it also resumes if paused?

Reply:
Makes sense. Re: the overlay, I was thinking that maybe we should just add a traditional pause menu with access to settings? Would that make more sense than having settings somewhere else?

Reply:
Yes, I think having a pause menu with access to the setting makes sense. However, I guess one should also be able to reach the settings upfront, probably coming from the title-stage. Maybe there is a way that we can just build a settings stage and step into it when pausing and resuming the main state afterwards?

Reply:
Yeah, I agree. Will see if I can improve this when I'm home during the evening.

Reply:
This looks like a basic appraoch for a pause menu: https://phaser.io/examples/v2/misc/pause-menu

Reply:
@paulgoetze Thanks. I'll have a look sometime tomorrow. Been busy with work and christmas stuff.

Reply:
:christmas_tree: cool! Shouldn’t be any kind of pressure, I just came across the link ;)

Reply:
@paulgoetze FYI: I'm back from the Christmas state of mind and will now have a look at this again